design-docs/README.md

```diff
@@ -108,12 +108,14 @@ with the bot.
 
 #### IOConfig
 
-From an implementation perspective, these will be configures as inputs/outputs
+From an implementation perspective, these will be configured as inputs/outputs
 (so cases like discord where one connection is both don't require two to be
 configured). These configurations will then present a list of Inputs and Outputs
 which can be used in Triggers.
 
 ```python
```

Comment:
Documentation Mismatch with Code : This does not appear to be a duplicate of any code anywhere in mewbot. Where should it be?

Reply:
It's demonstrative rather than actual code. pycharm assumes that python code blocks should be correct and happens to lint them as well, so I fixed them up whilst doing the refactoring.